sekoia_automation/connector/__init__.py

```diff
@@ -29,10 +41,44 @@ class DefaultConnectorConfiguration(BaseModel):
 
 
 class Connector(Trigger, ABC):
-    configuration: DefaultConnectorConfiguration
+    CONNECTOR_CONFIGURATION_FILE_NAME = "connector-configuration"
+    default_configuration: DefaultConnectorConfiguration
 
     seconds_without_events = 3600 * 6
 
+    @property
+    def configuration(self) -> dict | BaseModel | None:
```

Comment:
I think that a connector configuration can't be `None` or a `dict` but we must make sure of it by checking the existing modules.

```suggestion
    def configuration(self) -> DefaultConnectorConfiguration:
```